parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -167,6 +167,9 @@ ensureKubelet
 ensureJournal
 
 if [[ -n "${MASTER_NODE}" ]]; then
+    if dpkg --compare-versions ${KUBERNETES_VERSION} ge 1.16; then
```

Comment:
`dpkg` only works on ubuntu (I think)

Reply:
👍 I replaced it with a `version_lte` function that I tested on CoreOS as well as Ubuntu.